ebu_tt_live/node/delay.py

```diff
@@ -93,3 +123,62 @@ def is_explicitly_timed(element):
                 res = is_explicitly_timed(child.value)
                 if res:
                     return res
+
+
+class UntimedPathFinder(RecursiveOperation):
+
+    _path_found = False
+    _timed_element_stack = None
+
+    def __init__(self, root_element):
+        self._timed_element_stack = []
+        super(UntimedPathFinder, self).__init__(
+            root_element,
+            filter=lambda value, element: isinstance(value, TimingValidationMixin)
+        )
+
+    def _is_begin_timed(self, value):
+        if value.begin is not None:
+            print '{} begin={}'.format(value, value.begin)
```

Comment:
Probably don't need the print for the merge version? Looks like a debug statement unless I've misunderstood.